### DISCUSSION TITLE
no guide line

### DISCUSSION THREAD

**Discussion by chukwuemekaamblessedchinenye** [ChipaDevTeam/PocketOptionAPI#12] [General]
How can i install it and make use of it , like installing the requirement and other

**Comment by theshadow76** [comment]
Talk to us in the discord, as this is in development, there is no docs yet.

https://discord.com/invite/Ge2yXheP

**Reply by matrixhead21** [reply]
The discord link is not working . Please chech and reupload or add me on discord - tanmayvarma#4746

**Reply by theshadow76** [reply]
https://discord.gg/kaZ8uV9b6k
